test/gpu/ops_test.cpp

```diff
@@ -722,6 +723,52 @@ struct test_triadd_broadcast : verify_program<test_triadd_broadcast>
     }
 };
 
+struct test_gelu : verify_program<test_gelu>
+{
+    migraphx::program create_program() const
+    {
+        migraphx::program p;
+        std::vector<size_t> input_lens{1, 1, 5};
+        auto x            = p.add_parameter("x", {migraphx::shape::float_type, input_lens});
+        auto half         = p.add_literal(0.5f);
+        auto one          = p.add_literal(1.0f);
+        auto sqrt2        = p.add_literal(static_cast<float>(M_SQRT2));
```

Comment:
Can you add a test changing the values of the literals to something else? Maybe change `half` to `2.0` and `sqrt2` to be the sqrt(3). It doesn't need to fuse necessarily, but we should make sure it is correct.

Reply:
I will make my matcher more robust since the values for sqrt2 and half are not checked currently. Therefore, the test would fail.